2select-html/README.md

```diff
@@ -0,0 +1,3 @@
+![phillip zastrow logo](https://pbs.twimg.com/profile_images/1452633114044403715/d3liT5vd_400x400.jpg)
```

Comment:
This is a cute logo to include, but did you ask Phillip if you could use it here?

Reply:
No...but I will ask him! Honestly, I know better!
:::hangs head in shame::::

Reply:
Yay! I have permission. I'll remember this going forward--so important. :)